src/components/PersonListItem.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react'
+import { connect } from 'react-redux'
+// import { Link } from 'react-router'
+const { object, string } = React.PropTypes
+import Avatar from './Avatar'
+import Link from 'react-router'
+
+@connect(({ people }, { personId }) => {
+  console.log('Connect', people)
+  console.log('PersonId', personId)
+  return { person: people[personId] }
+})
+export default class PersonListItem extends React.Component {
+  static propTypes = {
+    person: object,
+    personId: string
```

Comment:
it's ok to just use `id` here instead of `personId` since its meaning is apparent from context